src/routes/jira/jira-router.ts

```diff
@@ -13,3 +13,15 @@ JiraRouter.use("/configuration", JiraConfigurationRouter);
 JiraRouter.post("/sync", JiraContextJwtTokenMiddleware, JiraSyncPost);
 JiraRouter.use("/events", JiraEventsRouter);
 JiraRouter.use("/select-version", JiraSelectVersionRouter);
+
+// TODO: remove this before merging this branch
```

Comment:
Is this still valid? Do we still need to remove this before merge?

Reply:
Nop, removing this...